src/db/tables/email_notifications.js

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import initKnex from "knex";
+import knexConfig from "../knexfile.js";
+const knex = initKnex(knexConfig);
+
+ /**
+  * @param {number} subscriberId
+  */
+ async function getAllEmailNotificationsForSubscriber(subscriberId){
+  console.info("getAllEmailNotificationsForSubscriber: ", subscriberId);
+  return await knex("email_notifications")
+    .where("subscriber_id", subscriberId)
+    .orderBy("id")
+    .returning("*");
+}
+
+/**
+ * @param {number} subscriberId
+ * @param {number} breachId
+ * @param {string} email
+ */
+async function getEmailNotification(
+  subscriberId,
+  breachId,
+  email
+){
+  console.info(
+    `getEmailNotification for subscriber: ${subscriberId}, breach: ${breachId}`,
+  );
+  const res = await knex("email_notifications")
```

Comment:
This looks good to me overall - do we want to think about `SELECT .. FOR UPDATE` to handle the case where there may be workers running in parallel (which I think we'll need to do in order to keep up with the backlog)?

Reply:
Good call.. Think we should add it for all gets? Like `getNotifiedSubscribersForBreach` below?

We aren't actually using some of these functions at the moment btw

Reply:
I think it would be safest, we can check the stats to see if it's causing problems but I doubt it will. If we're getting a lot of duplicates from upstream that is probably something we should follow up on anyway.